README.md

```diff
@@ -1,20 +1,21 @@
 # Kai AI Platform
+
```

Comment:
PR Template looks fine, include testing methodology and results. Then, include a request schema where you define the types for each input field